frontend/src/components/ConfirmDialog.tsx

```diff
@@ -1,36 +1,44 @@
-// ConfirmDialog.tsx  - material ui
+import CloseIcon from '@mui/icons-material/Close';
```

Comment:
I believe the only import changes I made were to remove the comment on the first line, fix the `CloseIcon` import, and import `React`. The rest of the changes are just alphabetizing.